tools/nnicmd/config_utils.py

```diff
@@ -71,3 +65,44 @@ def read_file(self):
             except ValueError:
                 return {}
         return {}
+
+class Experiment:
```

Comment:
if this class is for maintaining experiment list, why not rename `Experiment` to `Experiments`?